modules/loki-rate-limit-errors.adoc

```diff
@@ -0,0 +1,71 @@
+// Module is included in the following assemblies:
+// * logging/cluster-logging-loki.adoc
+// * logging/cluster-logging-external.adoc
+
+:_content-type: PROCEDURE
+[id="loki-rate-limit-errors_{context}"]
+= Troubleshooting Loki rate limit errors
+
+If the Log Forwarder API forwards a large block of messages to Loki that exceeds the rate limit, Loki generates rate limit (`429`) errors. You can fix this issue by modifying the `LokiStack` custom resource (CR).
+
+[IMPORTANT]
+====
+The `LokiStack` CR is not available on Grafana-hosted Loki. This topic does not apply to Grafana-hosted Loki servers.
+====
+
+.Conditions
+
+* The Log Forwarder API is configured to forward logs to Loki.
+
+* Your system sends a block of messages that is larger than 2 MB to Loki. For example:
++
+[source,text]
+----
+"values":[["1630410392689800468","{\"kind\":\"Event\",\"apiVersion\":\
+.......
+......
+......
+......
+\"received_at\":\"2021-08-31T11:46:32.800278+00:00\",\"version\":\"1.7.4 1.6.0\"}},\"@timestamp\":\"2021-08-31T11:46:32.799692+00:00\",\"viaq_index_name\":\"audit-write\",\"viaq_msg_id\":\"MzFjYjJkZjItNjY0MC00YWU4LWIwMTEtNGNmM2E5ZmViMGU4\",\"log_type\":\"audit\"}"]]}]}
+----
+
+* After you enter `oc logs -c collector`, the collector logs in your {ocp-logging} cluster show the following error messages:
++
+[source,text]
+----
+429 Too Many Requests Ingestion rate limit exceeded (limit: 8388608 bytes/sec) while attempting to ingest '2140' lines totaling '3285284' bytes
+----
++
+[source,text]
+----
+429 Too Many Requests Ingestion rate limit exceeded' or '500 Internal Server Error rpc error: code = ResourceExhausted desc = grpc: received message larger than max (5277702 vs. 4194304)'
+----
+
+* After you open the logs on the Loki server, error messages similar to the following are displayed:
++
+[source,text]
+----
+2023-08-25T16:08:49.301780Z  WARN sink{component_kind="sink" component_id=default_loki_infra component_type=loki component_name=default_loki_infra}: vector::sinks::util::retries: Retrying after error. error=Server responded with an error: 429 Too Many Requests internal_log_rate_limit=true
+----
+
+.Procedure
+
+* Update the `ingestionBurstSize` and `ingestionRate` spec in the `LokiStack` CR:
```

Comment:
I think we should provide some detail on how to properly configure these values. Just by reading the content we have, I didn't understand how to do it properly after reading the [official documentation](https://grafana.com/docs/loki/latest/configuration/#limits_config) and researching the algorithm used by Loki I think a good explanation would be something like the following.

`ingestionRate` configures the maximum size in MB of ingested samples that will be divided by the number of Distributor replicas. However, Loki is able to tolerate bursts of bigger samples in a single push request, this is done through`ingestionBurstSize`, so this value reflects the maximum sample size in MB that a single Distributor will be able to consume in a single push request.

Reply:
Added some additional info here, thank you!